server/mediapublic/models.py

```diff
@@ -24,9 +24,7 @@
 from zope.sqlalchemy import ZopeTransactionExtension
 import transaction
 
-DBSession = scoped_session(sessionmaker(
-    extension=ZopeTransactionExtension(),
-    expire_on_commit=False))
+DBSession = scoped_session(sessionmaker(expire_on_commit=False))
```

Comment:
So this is something I had to do on a few pyramid apps now, remove the ZopeTransactionExtension().  I have not yet been able to find solid docs that tell me that I have to use it, and the relationships never seem to work correctly when it's there ...